docs/prover/cli/conf-file-api.md

```diff
@@ -0,0 +1,100 @@
+Certora Prover Conf File API
+============================
+
+The **Conf File API** is an alternative API for  `certoraRun` . In terms of functionality 
+this API is identical to the [CLI Options](options.md) API. Instead of calling `certoraRun` 
+with a list of shell flags, some or all the flags can be stored in a [JSON](https://www.json.org/json-en.html) file 
+(to be more precise the format is [JSON5](https://json5.org/)):
+
+```bash
+certoraRun my_params.conf
+```
+<div style="background-color: #FFFFE0; padding: 10px; border: 1px solid #E6DB55;">
+To denote a text file as a conf file use the <strong>.conf</strong> suffix
+</div>
+
+Converting from CLI Options to JSON
+-----------------------------------
+
+**JSON Keys**
+
+JSON keys in the conf file are the CLI option flag names without the leading dashes:
+```
+The CLI flag --verify will be stored under the "verify" in the conf file
+```
+The JSON key for the input files in the CLI API is **"files"**
+
+**String Value CLI Options**
+
+Flags in CLI API that accept a single string will be stored as **JSON Strings**. Example:
+```
+"solc": "solc4.25"
+```
+**Number Value CLI Options**
+
+Flags in CLI API that accept numbers will be stored as **JSON Strings** not as **JSON Numbers**. Example:
+
+```
+"smt_timeout": "600"
+```
+
+**Boolean Value CLI Options**
+
+Since boolean flags in CLI API do not get a value they will be stored as **JSON true**. Example:
+
+```
+"send_only": true
+```
+**List Value CLI Options**
+
+Flags in CLI API that accept multiple strings will be stored as **JSON Arrays**. Example:
+```
+    "packages": [
+        "@balancer-labs/v2-solidity-utils=pkg/solidity-utils",
+        "@balancer-labs/v2-vault=pkg/vault"
+    ]
+```
+
+**Map Value CLI Options**
+
+Flags in CLI API that are maps will be stored as **JSON Objects**. Example:
+```
+    "solc_map": {
+        "A": "solc5.11",
+        "B": "solc5.9",
+        "C": "solc6.8"
+    }
+    
+```
+
+<div style="background-color: #FFFFE0; padding: 10px; border: 1px solid #E6DB55;">
```

Comment:
i like these yellow boxes! I do wonder if these should be
```
```{note}
content
```

```

instead? That seems to be what we do elsewhere in the doc when we have similar content.

Reply:
My intention was not to mark it as a note, more for emphasizing the text without breaking the flow. I removed the yellow boxes.

Reply:
i liked them :)